README.md

```diff
@@ -0,0 +1,174 @@
+<a name="readme-top"></a>
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍